### PR TITLE
LF-19381 updating sdk version for fycon changes

### PR DESCRIPTION
**Pull request by ktabors**
https://livefyre.atlassian.net/browse/LF-19381

### CODE REVIEW

**Comment by ktabors** [comment]
#review

File: src/styles/wall-view.css [hunk 1]
```diff
@@ -9,3 +9,7 @@
     -webkit-box-sizing: border-box;
     box-sizing: border-box;
     padding: 5px; }
+
+.streamhub-media-wall-view .content-tumblr .content-attachments-stacked {
```
Comment (markdoten) [inline src/styles/wall-view.css]: Shouldn't this style come in from the SDK?

**Comment by markdoten** [comment]
#feedback

File: src/styles/wall-view.css (hunk 1 above)
Comment (ktabors) [inline src/styles/wall-view.css]: it's does... this is a complied file with the change in it.

**Pull request merged**